mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.tmpl

```diff
@@ -414,12 +416,14 @@ API (for read pools, effective_availability_type may differ from availability_ty
 									"binary_log_enabled": {
 										Type:         schema.TypeBool,
 										Optional:     true,
+										Computed:     true,
```

Comment:
It's been a while, so I just want to confirm that it's okay to mark these as computed? Users won't be able to unset these fields if they're Computed.

If you're only doing this so that diff.Clear works, I'd recommend moving away from diff.Clear & using DiffSuppressFuncs instead.